### PR TITLE
[EDU-3964] feat: add real-time purge limits

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Add limits to real-time purge

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3964]: https://aziontech.atlassian.net/browse/EDU-3964?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/domains/domains.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ To enable environment selection, contact the [Sales Team](https://www.azion.com/
 
 ## CNAME
 
-CNAME is the standard service responsible for creating CNAME settings, enabling you to customize the edge application's  domain. Each domain can accommodate up to 50 CNAMEs.
+CNAME is the standard service responsible for creating CNAME settings, enabling you to customize the edge application's  domain.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/domains/domains.mdx:40]:
```suggestion
CNAME is the standard service responsible for creating CNAME settings, enabling you to customize the edge application's domain.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/real-time-purge.mdx [hunk 2]
```diff
@@ -186,13 +186,18 @@ When needed, you can query the purge history to filter the user who made the pur
...
-- **Wildcard** purges: up to 2,000 requests per day (per 24h interval), one wildcard URL per request. The wildcard URL is limited to 256 characters.
-- Purge history is limited to 6 months and up to 1 million requests.
-
-
+| Scope | Limit |
+| --- | --- |
+| Purge history time | 6 months |
+| Number of requests in history | 1,000,000 |
+| Wildcard size | 256 characters |
+| URL and Cache Key size | 4.096 characters |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/real-time-purge.mdx:194]:
```suggestion
| URL and Cache Key size | 4,096 characters |
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/real-time-purge.mdx [hunk 3]
```diff
@@ -186,13 +186,18 @@ When needed, you can query the purge history to filter the user who made the pur
...
+
+These are the **default limits** for each Service Plan:
+
+| Scope | Developer | Business, Enterprise, and Mission Critical |
+| --- | --- | --- | --- | --- |
+| Purge operations | 1,000 per month | 1,000 per month<br />Additional purge operations are negotiable |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/real-time-purge.mdx:200]: será que não vale também a tip de increase limits?

**Review by hannah-sk — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/real-time-purge.mdx [hunk 4]
```diff
@@ -186,13 +186,18 @@ When needed, you can query the purge history to filter the user who made the pur
...
+| Purge history time | 6 months |
+| Number of requests in history | 1,000,000 |
+| Wildcard size | 256 characters |
+| URL and Cache Key size | 4,096 characters |
+
+These are the **default limits** for each Service Plan:
+
+| Scope | Developer | Business, Enterprise, and Mission Critical |
+| --- | --- | --- | --- | --- |
+| Purge operations | 1,000 per month | 1,000 per month<br />Additional purge operations are negotiable |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/real-time-purge.mdx:200]:
```suggestion
| Purge operations | 1,000 per month | 1,000 per month<br />Additional purge operations are negotiable |

:::tip
**Increase limits** <br></br>
You can request to increase the limits based on your plan. Contact the [technical support team](/en/documentation/services/support/) to request it.
:::
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/real-time-purge.mdx [hunk 5]
```diff
@@ -188,13 +188,18 @@ Quando necessário, você pode consultar o histórico de purge para filtrar o us
...
+| Tempo de histórico de purge | 6 meses |
+| Número de requisições em histórico | 1.000.000 |
+| Tamanho de Wildcard | 256 caracteres |
+| Tamanho de URL e Cache Key | 4.096 caracteres |
+
+Estes são os **limites default** para cada Plano de Serviço:
+
+| Escopo | Developer | Business, Enterprise e Mission Critical |
+| --- | --- | --- | --- | --- |
+| Operações de purge | 1.000 por mês | 1,000 por mês<br />Operações adicionais de purge são negociáveis |
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/real-time-purge.mdx:203]:
```suggestion

:::tip
**Aumente limites** <br></br>
Você pode solicitar o aumento dos limites com base no seu plano. Contate o [time de suporte técnico](/pt-br/documentacao/servicos/suporte/) para fazer a solicitação.
:::

```

**Pull request merged**